lib/mutant/meta/example/dsl.rb

```diff
@@ -9,18 +8,20 @@ class DSL
         # Run DSL on block
         #
         # @return [Example]
-        def self.run(file, block)
+        def self.call(file, block)
           instance = new(file)
           instance.instance_eval(&block)
```

Comment:
I think the docs for `BasicObject#instance_eval` are incorrect then: http://ruby-doc.org/core-2.2.0/BasicObject.html#method-i-instance_eval